optaplanner-wb-webapp/pom.xml

```diff
@@ -825,6 +824,7 @@
     <dependency>
       <groupId>org.jboss.errai</groupId>
       <artifactId>errai-cdi-client</artifactId>
+      <!-- Unlike all other client only dependencies, errai-cdi-client must not be in scope provided -->
```

Comment:
@ge0ffrey @mbarkley and I will refactor Errai (4.0) so that errai-cdi-client is no longer required on the server.